spec/models/follow_spec.rb

```diff
@@ -73,7 +73,7 @@
     it "sends an email notification" do
       user_2.update_column(:email_follower_notifications, true)
       expect do
-        perform_enqueued_jobs do
+        Sidekiq::Testing.inline! do
```

Comment:
this helper is coming :D https://github.com/thepracticaldev/dev.to/pull/5326/files#diff-18f708453e8e0245d43e8dad3cee94e8R56

Reply:
let's see which one gets merged quicker^^. I can adjust to using the helper when cleaning up the ActiveJob model if this PR goes to master first. Else that is a quick adjustment.